### PR TITLE
Added V1 properties benchmarks data

### PR DESCRIPTION
**Pull request by tiagogmarques**
Data for stimuli and assemblies

### CODE REVIEW

**Comment by mschrimpf** [comment]
thanks this looks much better! The `lookup.db` is still there, can you please delete?

Also can you please add the identifiers of the assemblies to unit tests like in [this recent PR](https://github.com/brain-score/brainio_collection/pull/37/files) to make sure the assemblies can be retrieved and aren't lost? It would be even better if you add some assertions on assembly shape (like [here](https://github.com/brain-score/brainio_collection/blob/6b3e90183067e8d40b2bce1bc5ab08fb220b4367/tests/test_assemblies.py#L171-L179) and below) and stimulus set shape (like [here](https://github.com/brain-score/brainio_collection/blob/6b3e90183067e8d40b2bce1bc5ab08fb220b4367/tests/test_stimuli.py#L79-L82)), but I understand if it's not applicable in this case.

File: brainio_collection/packaging.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from tqdm import tqdm
 import numpy as np
 from PIL import Image
-import cv2
+#import cv2
```
Comment (mschrimpf) [inline brainio_collection/packaging.py:12]: undo this deletion

File: brainio_contrib/dicarlo/marques/marques_devalois1982b.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+def collect_data():
+    # Peak spatial frequency data
+    peak_spatial_frequency_bins = np.array([0.35, 0.5, 0.7, 1.0, 1.4, 2.0, 2.8, 4.0, 5.6, 8.0, 11.2, 16.0, 22.4])
+    # peak_spatial_frequency_simple_hist = np.array([0, 4, 4, 8, 25, 33, 26, 28, 12, 5, 2, 1])
+    # peak_spatial_frequency_complex_hist = np.array([0, 0, 0, 9, 9, 7, 10, 23, 12, 8, 3, 3])
+    # peak_spatial_frequency_hist = (peak_spatial_frequency_simple_hist + peak_spatial_frequency_complex_hist)
+    #
+    # peak_spatial_frequency = gen_sample(peak_spatial_frequency_hist, peak_spatial_frequency_bins, scale='log2')
```
Comment (mschrimpf) [inline brainio_contrib/dicarlo/marques/marques_devalois1982b.py:19]: dead code

File: brainio_contrib/dicarlo/marques/marques_gen_stim.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+               SIZE_STIM_NAME: SIZE_PARAMS}
+
+# POS_DICT = {RF_STIM_NAME: POS_rf, ORIENTATION_STIM_NAME: POS, SF_STIM_NAME: POS, SIZE_STIM_NAME: POS}
+# RADIUS_DICT = {RF_STIM_NAME: np.array([1/6]), ORIENTATION_STIM_NAME: np.array([0.25, 0.5, 1]),
+#                SF_STIM_NAME: np.array([0.75, 2.25]), SIZE_STIM_NAME: RADIUS}
+# SF_DICT = {RF_STIM_NAME: np.array([3]), ORIENTATION_STIM_NAME: np.array([1, 2, 4]), SF_STIM_NAME: SF,
+#            SIZE_STIM_NAME: np.array([1, 2, 4])}
+# ORIENTATION_DICT = {RF_STIM_NAME: ORIENTATION[[0, 3, 6, 9]], ORIENTATION_STIM_NAME: ORIENTATION,
+#                     SF_STIM_NAME: ORIENTATION[[0, 2, 4, 6, 8, 10]], SIZE_STIM_NAME: ORIENTATION[[0, 2, 4, 6, 8, 10]]}
+# PHASE_DICT = {RF_STIM_NAME: PHASE[[0, 4]], ORIENTATION_STIM_NAME: PHASE, SF_STIM_NAME: PHASE, SIZE_STIM_NAME: PHASE}
```
Comment (mschrimpf) [inline brainio_contrib/dicarlo/marques/marques_gen_stim.py:74]: dead code

**Review by mschrimpf — COMMENTED** [review]
I saw some pieces of dead code which might point to an underlying issue (it's not a big deal in packaging code in my mind, just want to point it out in case there's something else).

The main missing piece are the **unit tests** https://github.com/brain-score/brainio_collection/pull/46#issuecomment-713115136
> can you please add the identifiers of the assemblies to unit tests like in [this recent PR](https://github.com/brain-score/brainio_collection/pull/37/files) to make sure the assemblies can be retrieved and aren't lost? It would be even better if you add some assertions on assembly shape (like [here](https://github.com/brain-score/brainio_collection/blob/6b3e90183067e8d40b2bce1bc5ab08fb220b4367/tests/test_assemblies.py#L171-L179) and below) and stimulus set shape (like [here](https://github.com/brain-score/brainio_collection/blob/6b3e90183067e8d40b2bce1bc5ab08fb220b4367/tests/test_stimuli.py#L79-L82)), but I understand if it's not applicable in this case.

File: brainio_collection/packaging.py (hunk 1 above)
Comment (stothe2) [reply brainio_collection/packaging.py]: Yes, you're right (sorry, I forgot to delete it in the end)! Can you or Tiago do it here?

Comment (mschrimpf) [reply brainio_collection/packaging.py]: thanks!
@tiagogmarques can you delete the line, and also delete it from the [`setup.py`](https://github.com/brain-score/brainio_collection/blob/992ae550d38681843cabfc37509d540acc44c8f6/setup.py#L22)?

Comment (tiagogmarques) [reply brainio_collection/packaging.py]: I don't find it in the setup.py. Is it part of the opencv-python package?

Comment (stothe2) [reply brainio_collection/packaging.py]: Yep, it's opencv-python.

**Comment by mschrimpf** [comment]
ping @tiagogmarques

**Review by mschrimpf — APPROVED** [review]
@tiagogmarques if you approve of the tests, this now looks good to me. Feel free to add more though!

**Comment by mschrimpf** [comment]
@tiagogmarques we're getting ready to merge brainio_base and brainio_collection. Can you please check this PR asap and tell me if the tests I added look good or add additional ones as you see fit?

**Comment by tiagogmarques** [comment]
The tests are good and comprehensive. I don't think it's necessary to add anything else.

**Pull request merged**